color/color.go

```diff
@@ -0,0 +1,78 @@
+package multicolor
+
+import (
+	"fmt"
+	"strings"
+	"unicode"
+
+	"github.com/ghishadow/color"
+)
+
+// New returns a color attribute build from supplied attribute names.
+// This function will panic if invalid attributes are supplied.
+func New(attributes ...interface{}) *ColorWriter {
+	return &ColorWriter{Color: color.New(Attributes(attributes...)...)}
+}
+
+// TryNew returns a color attribute build from supplied attribute names.
+// If attributes cannot be converted into valid Attribute, an error is returned.
+func TryNew(attributes ...interface{}) (*ColorWriter, error) {
+	attr, err := TryConvertAttributes(attributes...)
+	return &ColorWriter{Color: color.New(attr...)}, err
+}
+
+// Set changes the current output color for all following output to stdout.
+// This function will panic if invalid attributes are supplied.
+func Set(attributes ...interface{}) *ColorWriter {
```

Comment:
This is just weird. You set a color to the writer and then all future writers have that color?
Mixing static and instance stuff is confusing in general, I think
IMO, it shouldn't create the writer at all. Just set the global color

Reply:
This is an override of the original color library. It is just there to be compliant with other attributes but the behaviour is originally from fatih.

Reply:
`Set` actually returns a color. Weird, OK